### ISSUE TITLE
[ValueCard] Tooltip flows of left side of the screen

### ISSUE THREAD

**Issue by stuckless** [IBM/carbon-addons-iot-react#1057]
### Describe the bug

Open the value card story that shows the no data.  click on the icon to render the tooltip.  It flows off the screen to the right.

Overall I think we stop using Carbon's tooltip since they are never going to create a tooltip that responds to where it is placed on the screen, and from their comments they don't believe they should mange this.  It means that right now every app developer that is using a carbon tooltip has to hope that their content isn't shown too close to the top, bottom, right, left of the screen (it works great if you center align all your conent in the middle of the screen, and or use a 20rem padding around your content to allow for the toolip).  It's a useless component, as is, and if we continue to use it, then it means that every component that we have to renders a tooltip will likely get a bug like this :(

This similar to issue #802 and might be related to #972 

Health is using the value card in a dashboard and it is likely that cars will be on left edgs of the screen with a "no data" warning.

### To Reproduce

![image](https://user-images.githubusercontent.com/1056553/78130797-f697bb00-73e7-11ea-982f-b97c2d04d0af.png)

**Comment by bjornalm** [comment]
@stuckless I agree with your analysis of the Carbon tooltip. What we temporarily could do in this case to mitigate the problem is to prevent the tooltip from opening directly under the icon.

So we can disable the icon tooltip completely for cards medium size and larger and only rely the user clicking on the header or text body which will also show the tooltip, or perhaps better include both the header and the icon as a tooltip trigger that shows the tooltip in the middle of the card.
